x-pack/solutions/observability/plugins/observability_ai_assistant_management/public/routes/components/knowledge_base_tab.tsx

```diff
@@ -222,6 +223,16 @@ export function KnowledgeBaseTab() {
     setQuery(e?.currentTarget.value || '');
   };
 
+  if (knowledgeBase.status.loading) {
```

Comment:
The spinner will now show while the knowledge base is installing every time it polls for status.  I think this is fine, though.